### PR TITLE
interfaces.xml Make the class property truly virtual

### PR DESCRIPTION
**Pull request by mmalferov**
In the footsteps of the issue: https://github.com/php/doc-en/issues/4151

### CODE REVIEW

File: language/oop5/interfaces.xml [hunk 1]
```diff
@@ -154,7 +154,10 @@ class C2 implements I
     // This example creates a virtual property, which is fine.
     public string $writeable {
         get => $this->written;
-        set => $value;
+
```
Comment (iluuu1994) [inline language/oop5/interfaces.xml:157]: Nit: I would prefer to minimize spaces.

**Review by iluuu1994 — APPROVED** [review]
Thank you! Btw, `$both` also looks suspicious. It only returns `$all`, but never writes to it. @Crell Can you clarify what the intention was here?

File: language/oop5/interfaces.xml (hunk 1 above)
Comment (mmalferov) [reply language/oop5/interfaces.xml:157]: Do you mean the empty line between the get and set hooks?

Comment (iluuu1994) [reply language/oop5/interfaces.xml:157]: Yes

**Comment by Crell** [comment]
Ha!  That example was copied directly from the RFC, so, oops?  Thanks for the catch.

**Pull request merged**